### DISCUSSION TITLE
[Feature] Page up/down Arrow key support

### DISCUSSION THREAD

**Discussion by ffchung** [immich-app/immich#14385] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Page up/down Arrow keyboard handle

People Page
Places Page

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by SoleInvictus** [comment]
PLEASE! As someone with RSI issues, mouse wheeling through hundreds of faces becomes painful. Simple page up/page down support would be amazeballs.

**Comment by rrrrrraa** [comment]
#23345 was closed as a duplicate of this, so could this feature request please be expanded to include support for Page Up/Down keys to navigate to next and previous photos on the photo and slideshow view on Web, as well as in photo view on Mobile/Android ?
